site/docs/transform/stack.md

```diff
@@ -94,8 +96,47 @@ If you want to define custom sort order, you can derive a new field using the [`
 
 Note: we plan to have [a better syntax for customized sort order](https://github.com/vega/vega-lite/issues/2915) in the future.
 
-## Layering Lines on top of Stacked Area Chart
+### Layering Lines on top of Stacked Area Chart
 
 Since `line` marks are not stacked by default, to layer lines on top of stacked area charts, you have to manually set the `stack` offset for the lines.
 
 <div class="vl-example" data-name="normalized/stacked_area_overlay_normalized"></div>
+
+{:#transform}
+## Stack Transform
+
+{: .suppress-error}
+```json
+// A View Specification
+{
+  ...
+  "transform": [
+    {"stack": ..., "groupby": ..., "offset": ..., "sort": ..., "as" ...} // Stack Transform
+     ...
+  ],
+  ...
+}
+```
+
+The `stack` transform in the `transform` array has the following properties:
+
+{% include table.html props="stack,groupby,offset,sort,as" source="StackTransform" %}
```

Comment:
Somehow the table is still empty

![image](https://user-images.githubusercontent.com/111269/41995827-da47473a-7a07-11e8-9a01-906984df0a4b.png)